guides/doc-Content_Management_Guide/topics/Importing_Content.adoc

```diff
@@ -451,8 +452,8 @@ Use this option if you have one of the following errors:
 . In the {Project} web UI, navigate to *Content* > *Products*.
 . Select the product containing the corrupted repository.
 . Select the name of a repository you want to synchronize.
-. From the *Select Action* menu, select *Advanced Sync*.
```

Comment:
I think this is still relevant, as optimized sync and complete sync are still accessible after clicking on Advanced sync. The option here refers to what the user might want to select, either optimized or complete. The section after this elaborates on the meaning of the different options. It probably might be better to add another section for "Verify Content Checksum" outside of the advanced sync section.